tools/deployment/make_linux_package.sh

```diff
@@ -117,6 +117,7 @@ function main() {
   platform OS
   distro $OS DISTRO
 
+  touch "$BUILD_DIR/$PACKAGE_NAME$(get_pkg_suffix)"
```

Comment:
The line right below this calls `realpath`, which fails if the file is not present (see my note about `realpath` on my PR description). I suppose that alternatively we can omit calling `realpath` alltogether, though I assumed it was there for some specific reason (like something needing fully qualified path and not a relative path?).